### ISSUE TITLE
Automatic Group Assignment

### ISSUE THREAD

**Issue by mlinton** [projectsend/projectsend#492]
When a client uploads a file I would like to automatically add the group category assignment to that file, so that all other members of the group get notified and have access to the file.

Is this possible?

**Comment by ilackskills** [comment]
Yes anything is possible with programming, if you are looking for someone to do that for you you may need to pay someone unless its something more people would use.

**Comment by bluesymbol** [comment]
This is something we will require realistically - we have end users uploading files, users are only ever members of one group. It's causing a massive admin headache to not have this feature. 

I've seen a few requests for this around now...

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
